docs/geedocs/5.3.0/answer/7160007.html

```diff
@@ -49,6 +49,16 @@ <h5>
       This can be done by configuring the <code>PURGE</code> and <code>PURGE_LEVEL</code> settings in <code>/etc/opt/google/XMLparams</code>. 
       This file must be created manually. View the <a href="../answer/176738.html">Configure Fusion performance</a> document for more details.</p>
     <p><strong>Added a <code>long_version</code> parameter to the SCons build. </strong>This will allow the user to override the long version string.</p>
+
+    <p><strong>Added PUBLISHER_ROOT size checking.</strong>
+       ASSET_ROOT checking is performed in fusion installer - not duplicated in server installer.
+       Added <code>-pgu pguser</code>, <code>-au apacheuser</code>, and <code>-g group</code> options.
+       Added prompt and summary output similar to fusion installer.</p>
+
+    <p><strong>We have improved the performance of large builds</strong> by consolidating some of the notifications that an asset's child or input has changed. By preventing (some) duplicate notifications we reduce the number of times each asset has to calculate its state, significantly improving performance.
+
+    <p><strong>Google has developed updated kmlrender libraries.</strong></p>
```

Comment:
Is this phrase incomplete ? It mentioned that Google has... but it does not state what is new in 5.3.0 as far as those kml libraries. Did you meant we have integrated a new version of kmlrender libraries ?

Reply:
I don't know - the PR that this refers to had very little information in it.  See https://github.com/google/earthenterprise/issues/1275.  I'll take any recommendations.

Reply:
The PR seems to refers to the use of a new/updated kml libraries used to render KML files in Portable. I assume the reason is performance and may be bug fixes @tst-dsheremata ?

Reply:
I was involved in that work so I can write up something when I get a few minutes.

Reply:
This might go better in the resolved issues section. The text would be something like this:

Description: Search results from KML layers do not display properly in the Earth Client.

Resolution: Updated KML rendering libraries with new versions provided by Google.